kaldb/src/test/java/com/slack/kaldb/logstore/search/LogIndexSearcherImplTest.java

```diff
@@ -1525,13 +1526,15 @@ public void testSearchOnlyHistogram() {
     assertThat(histogram.getBuckets().get(0).getDocCount()).isEqualTo(1);
     assertThat(histogram.getBuckets().get(1).getDocCount()).isEqualTo(1);
 
-    assertThat(
-            Long.parseLong(histogram.getBuckets().get(0).getKeyAsString()) >= time.toEpochMilli())
-        .isTrue();
-    assertThat(
-            Long.parseLong(histogram.getBuckets().get(1).getKeyAsString())
-                <= time.plusSeconds(10).toEpochMilli())
-        .isTrue();
+    DateTimeFormatter formatter = DateTimeFormatter.ISO_INSTANT;
```

Comment:
If I recall correctly, there was some difficulty in getting the return values here to work nicely with Grafana histogram rendering. We should verify that changing the response date format here isn't breaking the histogram rendering for Grafana.